### PR TITLE
Scrollview: rendering and style improvements

### PR DESCRIPTION
**Pull request by PreslavKozovski**

### CODE REVIEW

File: packages/bootstrap/scss/scrollview/_theme.scss [hunk 1]
```diff
@@ -1,35 +1,30 @@
 @include exports("scrollview/theme") {
 
-    .k-scrollview {}
+    //Remove these scoping selectors when the Angular team change the ScrollView rendering
```
Comment (elena-gancheva) [inline packages/bootstrap/scss/scrollview/_theme.scss]: Log an issue in `kendo-angular-scrollview` repo and link it here in the comment - for reference: https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/notification/_layout.scss#L66

File: packages/bootstrap/scss/_variables.scss [hunk 2]
```diff
@@ -1029,6 +1029,28 @@ $dialog-stretched-button-padding-y: $button-padding-y !default;
...
+$scrollview-pagebutton-primary-bg: null !default;
+$scrollview-pagebutton-primary-border: null !default;
+$scrollview-pagebutton-shadow: 0 0 2px rgba(0, 0, 0, .08) !default;
+
+$scrollview-pager-offset: 20px !default;
+$scrollview-pager-item-spacing: 20px !default;
+$scrollview-pager-item-border-width: 0 !default;
+$scrollview-pager-height: add-three($scrollview-pagebutton-size, 2 * $scrollview-pager-item-border-width, $scrollview-pager-item-spacing / 2 ) !default;
+
+$scrollview-navigation-size: 4.5em !default;
```
Comment (elena-gancheva) [inline packages/bootstrap/scss/_variables.scss]: This is not very descriptive name, why did you change the previous ` $scrollview-arrow-icon-size`?

File: packages/default/scss/scrollview/_layout.scss [hunk 3]
```diff
@@ -1,154 +1,175 @@
...
-        pointer-events: all;
-    }
+        .k-scrollview-pageable > li.k-button,
+        .k-scrollview-nav > li.k-link {
+            margin: 0 $scrollview-pager-item-spacing;
+            padding: 0;
+            width: $scrollview-pagebutton-size;
+            height: $scrollview-pagebutton-size;
+            box-sizing: content-box;
+            vertical-align: middle;
```
Comment (elena-gancheva) [inline packages/default/scss/scrollview/_layout.scss]: This align style is not applied here, move it for jquery rendering styles instead

File: packages/default/scss/scrollview/_layout.scss [hunk 4]
```diff
@@ -1,154 +1,175 @@
...
+            display: table;
+            position: absolute;
+            padding: 0;
+            height: 60%;
+            top: 20%;
+            text-decoration: none;
+            user-select: none;
+            cursor: pointer;
+            overflow: hidden;
+            -webkit-tap-highlight-color: $rgba-transparent;
```
Comment (elena-gancheva) [inline packages/default/scss/scrollview/_layout.scss]: [suggestion] consider exposing a variable for customization of this color, if not needed, ignore this comment

File: packages/default/scss/scrollview/_theme.scss [hunk 5]
```diff
@@ -1,49 +1,50 @@
...
+    kendo-scrollview.k-scrollview-wrap,
+    div.k-scrollview {
 
-    .k-scrollview-elements {
-        color: $scrollview-navigation-color;
-    }
+        //refactor when https://github.com/telerik/kendo-angular-scrollview/issues/130 is done
+        .k-scrollview-pageable > li.k-button,
+        .k-scrollview-nav > li.k-link {
+            @include fill(
```
Comment (elena-gancheva) [inline packages/default/scss/scrollview/_theme.scss]: Do we need those `fill` functionality to apply only `background` and `border` style? @joneff

File: tests/visual/scrollview.html [hunk 6]
```diff
@@ -13,8 +13,8 @@
 <body>
 
     <div id="test-area" class="grid">
```
Comment (elena-gancheva) [inline tests/visual/scrollview.html]: Add an Angular `Scrollview` visual test with the current rendering. Notify with comment which one is Angular and which is jQuery markup.

File: packages/default/scss/scrollview/_layout.scss [hunk 7]
```diff
@@ -1,154 +1,175 @@
...
+                left: 0;
+            }
         }
 
-        & > li {
-            display: inline-block;
-            overflow: hidden;
+        .k-scrollview-pageable,
+        .k-scrollview-nav {
+            margin: 0 auto;
```
Comment (elena-gancheva) [inline packages/default/scss/scrollview/_layout.scss]: Only `margin: 0;`. You already have `justify-content: center;` so no need to apply `auto`.

**Comment by elena-gancheva** [comment]
Also, please change the commit tag to `fix`

File: tests/visual/scrollview.html (hunk 6 above)
Comment (elena-gancheva) [reply tests/visual/scrollview.html]: also, add a TODO comment in the visual test for Angular to be updated when we refactor the component's rendering

File: packages/default/scss/scrollview/_theme.scss (hunk 5 above)
Comment (joneff) [reply packages/default/scss/scrollview/_theme.scss]: `@fill` will make sure that if any of the variables passed are `null` there won't be any output.
